plugins/pulp_rpm/plugins/distributors/iso_distributor/distributor.py

```diff
@@ -61,18 +61,19 @@ def publish_repo(self, transfer_repo, publish_conduit, config):
         repo = transfer_repo.repo_obj
         return super(ISODistributor, self).publish_repo(repo, publish_conduit, config)
 
-    def unpublish_repo(self, repo, config):
+    def unpublish_repo(self, transfer_repo, config):
```

Comment:
Is this what the superclass calls the parameter? Assuming no, I think it should be changed to match the superclass. The convention I've seen elsewhere has been to use the variable name ``repo_obj`` for the new-style repo object, so you could use that or something similar on line 76 below.